doc/quicklook.rst

```diff
@@ -189,3 +162,5 @@ MOSFIRE QL Output
 +++++++++++++++++
 
 If all goes smoothly, a ginga window will open with the resulting image.
+
```

Comment:
crufty white space

Reply:
I can remove it if it's an issue, but I tend to prefer having a couple of empty lines at the end of code files.

Reply:
if it's purposeful, it's totally fine.